lib/document-builder.ts

```diff
@@ -136,34 +137,34 @@ export class DocumentBuilder {
     return this;
   }
 
-  public addApiKey(
-    options: Optional<SecuritySchemeObject, 'type'> = {},
-    name = 'api_key'
+  public addBasicAuth(
+    options: Partial<Omit<HttpSchemeObject, 'type'>> = {},
+    name = 'basic'
```

Comment:
Making the change I mentioned above here so it would be
```suggestion
		name = 'basic',
		options: Partial<Omit<HttpSchemeObject, 'type'>> = {}
```

Reply:
@jmcdo29, Yes, this can also be added, but is it better in a minor update?..